### ISSUE TITLE
embedding_features argument in Pool

### ISSUE THREAD

**Issue by antipisa** [catboost/catboost#1812]
Problem:
Can you provide some explanation for the embedding_features argument to catboost.Pool? It's not available online, and the release notes only mentions this:

> Experimental functionality
> Supported embedding features as input and linear discriminant analysis for embeddings preprocessing. Try adding your embeddings as new columns with embedding values array in Pandas.Dataframe and passing corresponding column names to Pool constructor or fit function with embedding_features=['EmbeddingFeaturesColumnName1, ...] parameter. Another way of adding your embedding vectors is new type of column in Column Description file NumVector and adding semicolon separated embeddings column to your XSV file: ClassLabel\t0.1;0.2;0.3\t....

catboost version: 0.26.2
Operating System: Linux

i.e, are embedding_features the feature_names of numeric vectors, or are they nested arrays of user defined bins for numerical vectors? How does this work if data is a pandas.Dataframe?

**Comment by andrey-khropov** [comment]
Sorry, we will update the docs for `embedding_features`.

> are embedding_features the feature_names of numeric vectors

Basically, yes.

Embedding features are special features that have values as arrays of numerical values of certain length (see https://en.wikipedia.org/wiki/Word_embedding although their usage is not necessarily limited to NLP).

> How does this work if data is a pandas.Dataframe?

Values of columns marked as 'embedding_features' must have arrays as elements.

For example:

``py
import pandas as pd

import catboost as cb

features_df = pd.DataFrame({
    'f0': [0, 1, 2],
    'embedding1': [[0.0, 1.1, 0.0], [1.0, 2.1, 0.0], [0.11, 0.2, 0.3]]
})
label = [0, 1, 1]

pool = cb.Pool(features_df, label=label, embedding_features=['embedding1'])

model = cb.CatBoostClassifier(iterations=10)
model.fit(pool)

``

**Comment by zghonda** [comment]
Hello, 

Can you please share some information about the release of 0.26.2 ? 

For example when can we expect to see it in pypi. 

Thank you in advance.

**Comment by dbulgakov7** [comment]
Please add a description and usage examples of the use of embedding_features to the appropriate section of the documentation: https://catboost.ai/en/docs/concepts/python-reference_pool
Now there is nothing written at all about the embedding_features parameter.

**Comment by andrey-khropov** [comment]
Done in e95227bb357eef6319aaecd1e0fc863e438ae91b.
Examples: https://catboost.ai/en/docs/concepts/python-usages-examples#class-with-array-like-data-with-numerical,-categorical-and-embedding-features, https://catboost.ai/en/docs/concepts/python-usages-examples#with-numerical,-categorical-and-embedding-features
